src/less/cta-button/cta-button.less

```diff
@@ -67,13 +71,14 @@ a.cta-btn--large {
 
 a.cta-btn--large-truncated {
     .btn-truncate();
-
+    border-radius: calc(@button-height-large / 2);
```

Comment:
Let's avoid `calc` whenever we can since it has a performance cost. Let's create a new var in `button-mixins.less`, something like `@button-border-radius-large: 24px` and use the var here.